README.md

```diff
@@ -182,6 +197,36 @@ $googleAdsClient = (new GoogleAdsClientBuilder())
 Once you have an instance of `GoogleAdsClient`, you can obtain a service client
 for a particular service using one of the `get...ServiceClient()` methods.
 
+## Protobuf
+
+[Protobuf](https://developers.google.com/protocol-buffers/docs/overview) is used by
+[gRPC](https://grpc.io/about/), a core dependency of the Google Ads API for request and
+response transport.
+
+There are two implementations available for PHP
+
+1. C (native): for better performance
+1. PHP (pure): to get started easily
+
+By default implementation of Protobuf used is C. This might not be suitable for some users
+who then need to use the PHP implementation instead. The PHP implementation
```

Comment:
Should we provide some examples of use cases where the PHP implementation is needed?

Reply:
The main reason is security but there might be others. @fiboknacky do you have any good resources to share regarding this?

Reply:
I cannot think of other good usages too.
The main reason would be simplicity and in some web servers, it's probably hard to install C implementation, which requires you to install via sudo

Reply:
Ok, I am using the security example here then, it does not have to be exhaustive anyway.